tide/master/localstreamer/PixelStreamerLauncher.cpp

```diff
@@ -74,14 +77,20 @@ PixelStreamerLauncher::PixelStreamerLauncher( PixelStreamWindowManager& windowMa
              Qt::QueuedConnection );
 }
 
-void PixelStreamerLauncher::openWebBrowser( const QPointF pos, const QSize size,
+void PixelStreamerLauncher::openWebBrowser( QPointF pos, const QSize size,
                                             const QString url )
 {
     static int webbrowserCounter = 0;
     const QString& uri = QString( "WebBrowser_%1" ).arg( webbrowserCounter++ );
 
     const QSize viewportSize = !size.isEmpty() ? size : WEBBROWSER_DEFAULT_SIZE;
-    _windowManager.openWindow( uri, pos, viewportSize, true );
+    const qreal x = 0.5 * _config.getTotalWidth();
```

Comment:
this code duplicated in openWhiteboard(). Make it to a function:
QPointF PixelStreamerLauncher::_getDefaultWindowPosition() const
{
    return { 0.5 * _config.getTotalWidth(), 0.35 * _config.getTotalHeight() };
}